Dockerfile

```diff
@@ -50,6 +50,7 @@ COPY --from=eclipse-temurin:8-jdk-jammy /opt/java/openjdk /usr/lib/jvm/8
 COPY --from=eclipse-temurin:11-jdk-jammy /opt/java/openjdk /usr/lib/jvm/11
 COPY --from=eclipse-temurin:17-jdk-jammy /opt/java/openjdk /usr/lib/jvm/17
 COPY --from=eclipse-temurin:21-jdk-jammy /opt/java/openjdk /usr/lib/jvm/21
+COPY --from=openjdk:25-jdk-bookworm /usr/local/openjdk-25 /usr/lib/jvm/25
```

Comment:
I would add a comment to switch to `eclipse-temurin:25-jdk-jammy` by the end of Sebtember to not forget about it 😅

Reply:
Oh sure yes 👌  They moved to the next Ubuntu release in the meantime.